library/leakcanary-android/src/main/java/com/squareup/leakcanary/internal/Ui.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2015 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.leakcanary.internal;
+
+import android.content.res.Resources;
+import android.graphics.PorterDuffXfermode;
+import android.util.DisplayMetrics;
+
+import static android.graphics.PorterDuff.Mode.CLEAR;
+
+final class Ui {
```

Comment:
Should probably rename to a longer / less used name. Otherwise it's going to show up in everybody's class list when they type "Ui" if their project uses LeakCanary, and most likely they don't care about that class. Need to confirm though, not sure if IJ is smart enough to ignore package private classes.